numbers/src/test/java/org/dicio/numbers/lang/fr/PronounceNumberTest.java

```diff
@@ -0,0 +1,240 @@
+        assertEquals("quinzième", pf.pronounceNumber(15).shortScale(F).ordinal(T).get());
+        assertEquals("vingtième", pf.pronounceNumber(20).shortScale(T).ordinal(T).get());
+        assertEquals("vingtième", pf.pronounceNumber(20).shortScale(F).ordinal(T).get());
+        assertEquals("vingt-septième", pf.pronounceNumber(27).shortScale(T).ordinal(T).get());
+        assertEquals("vingt-septième", pf.pronounceNumber(27).shortScale(F).ordinal(T).get());
+        assertEquals("trentième", pf.pronounceNumber(30).shortScale(T).ordinal(T).get());
+        assertEquals("trentième", pf.pronounceNumber(30).shortScale(F).ordinal(T).get());
+        assertEquals("trente-troisième", pf.pronounceNumber(33).shortScale(T).ordinal(T).get());
+        assertEquals("trente-troisième", pf.pronounceNumber(33).shortScale(F).ordinal(T).get());
+        assertEquals("centième", pf.pronounceNumber(100).shortScale(T).ordinal(T).get());
+        assertEquals("centième", pf.pronounceNumber(100).shortScale(F).ordinal(T).get());
+        assertEquals("millième", pf.pronounceNumber(1000).shortScale(T).ordinal(T).get());
+        assertEquals("millième", pf.pronounceNumber(1000).shortScale(F).ordinal(T).get());
+        assertEquals("dix-millième", pf.pronounceNumber(10000).shortScale(T).ordinal(T).get());
+        assertEquals("dix-millième", pf.pronounceNumber(10000).shortScale(F).ordinal(T).get());
+        assertEquals("deux-millième", pf.pronounceNumber(200).shortScale(T).ordinal(T).get());
+        assertEquals("deux-millième", pf.pronounceNumber(200).shortScale(F).ordinal(T).get());
+        assertEquals("huit-mille-quatre-vingtième", pf.pronounceNumber(18691).ordinal(T).shortScale(T).get());
+        assertEquals("huit-mille-quatre-vingtième", pf.pronounceNumber(18691).ordinal(T).shortScale(F).get());
+        assertEquals("mille-cinq-cent-soixante-septième", pf.pronounceNumber(1567).ordinal(T).shortScale(T).get());
+        assertEquals("mille-cinq-cent-soixante-septième", pf.pronounceNumber(1567).ordinal(T).shortScale(F).get());
+
+        // big numbers
+        assertEquals("dix-huit-millionième", pf.pronounceNumber(18000000).ordinal(T).get());
+        assertEquals("dix-huit-mille-centième", pf.pronounceNumber(18000100).ordinal(T).get());
+        assertEquals("cent-vingt-sept-milliardième", pf.pronounceNumber(127000000000.0).ordinal(T).shortScale(T).get());
+        assertEquals("deux-cent-un-millionième", pf.pronounceNumber(201000000000.0).ordinal(T).shortScale(F).get());
+        //TODO: Check this for french correctness as well as short/long scale issues (billion/billard)
+        assertEquals("neuf-cent-treize-milliard-quatre-vingt-million-six-cent-mille-soixante-mille-cent-soixante-quatrième", pf.pronounceNumber(913080600064.0).ordinal(T).shortScale(T).get());
+        assertEquals("neuf-cent-treize-mille-quatre-vingt-million-six-cent-mille-soixante-mille-soixante-quatrième", pf.pronounceNumber(913080600064.0).ordinal(T).shortScale(F).get());
```

Comment:
I'm not sure you can repeat several times in a row "mille"

Reply:
I think it should be "neuf-cent-treize-milliard-quatre-vingt-million-six-cent-mille-soixante-quatrième"? "mille million" isn't very common is it? (Though saying 913080600064th itself isn't very common -_-')